common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -102,10 +102,42 @@
     <%
         body = capture(caller.body)
         body_dict = json.loads(body)
-        body_dict['lang'] = lang
+        locale = body_dict['lang']
+        ietf_to_transifex = {
+            'fr': 'fr_CA',
+            'zh-CN': 'zh_CN',
+            'es-419': 'es_419',
+            'ar': 'ar'
+        }
+
+        messages = "{}"
+        try:
+            messages_path = "{base}/studio-frontend/dist/i18n/messages/{locale}.json".format(
+                base=settings.STATIC_ROOT,
+                locale=ietf_to_transifex[locale]
+            )
+            with open(messages_path) as inputfile:
+                messages = inputfile.read()
+        except:
+            messages = "{}"
+
+        localeData = ""
+        try:
+            localeData_path = "{base}/studio-frontend/dist/i18n/locale_data/{locale}.json".format(
+                base=settings.STATIC_ROOT,
+                locale=locale.split('-')[0]
```

Comment:
the locale scripts and message data files are named differently, that's why these 2 similar paths format `locale` slightly differently.